rfc-149-switch-to-per-page-asset-loading.md

```diff
@@ -1,7 +1,9 @@
 ---
-Author: Ian James
-Date: 2 August 2022
-Deadline for decision: 16 August 2022
+status: accepted
+implementation: done
+status_last_reviewed: 2024-03-04
+status_notes: >
```

Comment:
Will having the `>` do anything odd in the CSV?

Reply:
No, it's YAML syntax for a Folded Block Scalar String. So long as it's parsed properly as YAML, the `>` character won't appear in the output. More info on YAML's disgustingly overkill multiline string handling: https://yaml-multiline.info/